settings.py

```diff
@@ -46,6 +46,7 @@
 
     # Third party
     'django_extensions',
+    'south',
```

Comment:
Yes, but this PR is just upgrading DRF, not Django.  There is another branch for Django 1.8 in this repo that switches to Django migrations: https://github.com/edx/edx-submissions/pull/23